plugins/prometheus_plugin/metrics.hpp

```diff
@@ -259,6 +259,7 @@ struct catalog_type {
       net_usage_us_produced_block.Increment(metrics.net_usage_us);
 
       update(produced_metrics, metrics);
+      update(speculative_metrics, metrics);
```

Comment:
it's okay that this increments speculative `num_blocks_created` right?

Reply:
Yes, I think so. If we want to report produced blocks also as speculative, seems we should report all the values.